docs/production/email-gateway.md

```diff
@@ -29,6 +29,15 @@ polling option is convenient for testing/developing this feature
 because it doesn't require a public IP address or setting up MX
 records in DNS.
 
+```eval_rst
+   .. note::
+       The incoming emails are rate-limited, and lay out the current limits:
+       
```

Comment:
Can you take out this whitespace ? (Also causes the tests to fail)

Reply:
I want to add a space between that two lines(line above and below the blank line) but I am not able to do it. I have tried some solutions like `<br>`, `\`, `&nbsp;` but it does not work, only that whitespace(used in PR) works but it fails tests. Do you have any suggestion here?

Reply:
How does it render if this line is removed?

Also, how does it render if this line is just a newline, without the other whitespace?

Reply:
* If the blank line is removed, the statement above and below the blank line is treated as continuous statement.
It looks like:
![lineremoved](https://user-images.githubusercontent.com/59444243/110197121-68d21480-7e6f-11eb-8e8f-c5d026be6d87.png)

* If the whitespace is removed, it looks like:
![whitespaceremoved](https://user-images.githubusercontent.com/59444243/110197140-88693d00-7e6f-11eb-82c5-36955389a893.png)
(Also it passes all the test cases)